.travis.yml

```diff
@@ -0,0 +1,176 @@
+os: linux
+dist: xenial
+sudo: required
+language: java
+jdk:
+  - openjdk11
+  
+addons:
+  firefox: "76.0.1"
+  
+services:
+  - xvfb
+  - docker
+
+  
+cache:
+  directories:
+    - $HOME/.m2
+# the cache can grow constantly
+before_cache:
+  - rm -rf $HOME/.m2/repository/org/alfresco/alfresco-share*
+  
+branches:
+  only:
+    - master
+    - /support\/.*/
+    - fix/Travis-share-build
+
+stages:
+  - name: build
+  - name: test
+    if: commit_message !~ /\[skip tests\]/
```

Comment:
In order to improve the mnemonic on commands, please use a naming convention similar to maven. eg: from [skip tests] to [skip-tests]